pkg/iptables/portmap.go

```diff
@@ -65,6 +65,12 @@ func ForwardPorts(ctx context.Context, tracker tracker.Tracker, updateInterval t
 			}
 		}
 
+		select {
+		case <-ctx.Done():
+			return nil
+		default: // continue the loop
```

Comment:
Would it make sense to put all [these](https://github.com/rancher-sandbox/rancher-desktop-agent/pull/51/files#diff-3e80c0cb8d6039a801e97c138b46435da1504ec5274ae33283a1fbbce2a7dcd4R27-R66) in the default case?

Reply:
That can work, but I don't see any reason to add a level of indent (given that the `Done` case returns immediately, and there are no other cases).

The `select` basically exists right now so we don't block on waiting for `Done` to happen (which would be silly).

If you think that makes the code easier to read, though, I'm happy to do that.